docs/zms_client.md

```diff
@@ -81,19 +81,13 @@ Use the `-z` option to point to the required environment for executing commands.
 ## How the ZMS Client Authenticates
 -----------------------------------
 
-The Athenz ZMS server requires the user to provide its `UserToken` to
-authorize the requests. The `UserToken` is obtained from the ZMS Server
+The Athenz ZMS server requires the user to authenticate using X509 certificate
+and private key. The certificate and private key are obtained from the ZMS Server
 with the user’s credentials and is valid for one hour.
 
-Before communicating with the ZMS Server, the `zms-cli` utility:
-
-1.  prompts for the user's password
-2.  generates an Authorization header and passes it to ZMS to retrieve a `UserToken`
-
 After communicating with the ZMS Server, the `zms-cli` utility caches
-the user's NToken in the user's home directory in the file `.ntoken`.
-It will continue to use the token until it expires, and then will prompt
-for the user's password.
+the user's certificate and private key are stored in $HOME/.athenz directory with key and cert filenames. 
```

Comment:
Again this is only specific to VM and will not be valid for opensource. The ZMS authorization is based on the configured authority.